src/GafferSceneModule/TweaksBinding.cpp

```diff
@@ -57,20 +57,6 @@ TweakPlugPtr constructUsingData( const std::string &tweakName, IECore::ConstData
 	return new TweakPlug( tweakName, tweakValue.get(), enabled );
 }
 
-class TweakPlugSerialiser : public PlugSerialiser
```

Comment:
I don't recall any good reason for needing this, so if all the tests are passing still let's call it a good 'un.